stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/adapter/viewholder/SwipeViewHolder.kt

```diff
@@ -156,6 +156,6 @@ public abstract class SwipeViewHolder(itemView: View) : BaseChannelListItemViewH
     }
 
     private companion object {
-        private val SWIPE_THRESHOLD = 16.dpToPxPrecise()
+        private fun swipeThreshold() = 16.dpToPxPrecise()
```

Comment:
I wonder why we cannot keep it here, cause it's Float primitive.
So in the end its gonna be converted into:
```
private static final float SWIPE_THRESHOLD = IntKt.dpToPxPrecise(16);
```

Reply:
Moved as class variable.